analyses/molecular-subtyping-ATRT/ATRT_subtyping.R

```diff
@@ -0,0 +1,64 @@
+#This script assigns ATRT into three known subtypes using methylation result.
+#Subtypiong esults is saved as ATRT_subtype_methylation.tsv
+
+# Set up library
+library(tidyverse)
+
+# Detect the ".git" folder -- this will in the project root directory.
+# Use this as the root directory to ensure proper sourcing of functions no
+# matter where this is called from
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# Set path to result
+results_dir <-
+  file.path(root_dir, "analyses", "molecular-subtyping-ATRT", "results")
+
+
+# Read in histologies
+histo <- 
+  readr::read_tsv(file.path(root_dir, "data", "histologies.tsv"), guess_max = 10000)
```

Comment:
```suggestion
histo <- 
  readr::read_tsv(file.path(root_dir, "data", "histologies-base.tsv"), guess_max = 10000)
```
use base in subtyping